src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,64 @@
+import {
+  CSSTransition,
+  TransitionGroup,
+} from 'react-transition-group';
```

Comment:
it should be in the package.json dependencies: `"react-transition-group": "^4.4.5"` and corresponding @types in devDependencies